ohttp/Cargo.toml

```diff
@@ -23,12 +25,20 @@ byteorder = "1.3"
 chacha20poly1305 = {version = "0.8", optional = true}
 hex = "0.4"
 hkdf = {version = "0.11", optional = true}
-hpke = {version = "0.10.0", optional = true, default-features = false, features = ["std"]}
 lazy_static = "1.4"
 log = {version = "0.4.0", default-features = false}
 rand = {version = "0.8", optional = true}
 sha2 = {version = "0.9", optional = true}
 thiserror = "1"
+hpke = {version = "0.10.0", optional = true, default-features = false, features = ["std", "x25519"]}
+
+[dependencies.hpke-pq]
+package="hpke"
+git="https://github.com/bwesterb/rust-hpke"
+branch="xyber768d00"
```

Comment:
Should we pin this to a specific commit?

Reply:
Done.